_includes/blog/post_info.liquid

```diff
@@ -1,5 +1,5 @@
 {% assign author = site.data.authors[include.author] %}
-{% assign date = include.date | default: "today" | date: "%B %-d, %Y" %}
+{% assign date = include.date | default: "today" | date: site.data.language.str_date_format %}
```

Comment:
What if `site.data.language.str_date_format` is not populated?
For retro-compatibility, if we can have something like
`{ assign date_format = site.data.language.str_date_format | default: '%B %-d, %Y' }` that would be great.